File: answers/Shobhit403/DAY 22/Answer2.py
S=int(input("Enter the Size: "))
R=int(input("Enter the position from where the number to be reversed: "))
z=[]
print("Enter the Elements: ")
for i in range(S):
    z.append(int(input()))
i=0
while i<R//2:
    v=z[i]
    z[i]=z[R-1-i]
    z[R-1-i]=v
    i+=1
print(z)

File: answers/Fab-Gaurav/Day 22/Answer_2.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int size, n, rev[size], arr[size];
    cout<<"Enter the total number of element : ";
    cin>>size;
    cout<<"Enter the elements : ";
    for(int i=0; i<size; i++){
        cin>>arr[i];
    }
    
    cout<<"Enter the position from where reverse has to be done : ";
    cin>>n;

    for(int i=0; i<size; i++){
        if(i<n){
           rev[i]=arr[n-(1+i)];
        }
        else{ rev[i]=arr[i];}
    }
    
    cout<<"After reversing : ";
    for(int i=0; i<size; i++){
        cout<<rev[i]<<", ";
    }
    return 0;
}
